bin/cylc-get-host-metric

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2018 NIWA
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""cylc get-host-metric [OPTIONS]
+
+   Get metrics for localhost, in the form of a JSON structure with
+   top-level keys as requested via the OPTIONS:
+
```

Comment:
Should the command name be `get-host-metrics` (plural)?  ... indulging in a bit of pendantry, sorry :grimacing: